examples/charts/unauthenticated/README.md

```diff
@@ -17,10 +17,10 @@ This sample shows how to use the JavaScript Embedding SDK to render unauthentica
 - Render an embedded chart on a web page
 - Set the charts theme to either `'light'` or `'dark'`
 - Get the charts current theme
-- Set the charts refresh interval
-  - Note, the default interval is 0, implying that after rendering, the chart will never refresh.
-  - The minimum refresh interval is 10 seconds.
-- Get the charts current refresh interval
+- Set the charts max data age
+  - Note, the max data age is 0, implying that after rendering, the chart will never refresh.
+  - The minimum max data age is 10 seconds.
```

Comment:
This is incorrect I think and is kinda contradicting with the first item saying it can be `0`. Max data age can be less than 10, the minimum autorefresh interval is 10s.